Sources/Plasma/FeatureLib/pfPython/plPythonParameter.h

```diff
@@ -377,9 +377,9 @@ typedef struct plPythonParameter
                 if ( count != 0 )
                 {
                     ST::char_buffer str;
-                    char *buffer = str.create_writable_buffer(count-1);
-                    stream->ReadLE(count, buffer);
-                    buffer[count-1] = 0;
+                    str.allocate(count - 1);
+                    stream->ReadLE(count - 1, str.data());
+                    (void)stream->ReadByte();
```

Comment:
Well, there goes my evil corrupted PRPs with non-null terminated plPythonParameter strings...

Reply:
It was actually protected in the old version as well, with the `buffer[count-1] = 0` line...

Reply:
Well I'll be dawg.